core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -350,6 +350,17 @@
   MASTER_METADATA_SUSPENDABLE("master.metadata.suspendable", "false", PropertyType.BOOLEAN,
       "Allow tablets for the " + MetadataTable.NAME
           + " table to be suspended via table.suspend.duration."),
+  MASTER_STARTUP_TSERVER_AVAIL_MIN_COUNT("master.startup.tserver.avail.min.count", "0",
+      PropertyType.COUNT,
+      "Minimum number of tservers that need to be registered before master will "
+          + "start tablet assignment - checked when master gets lock. To disable check, set to <= 0. "
+          + "Default is 0 (disabled). Added with version 1.9.4"),
+  MASTER_STARTUP_TSERVER_AVAIL_MAX_WAIT("master.startup.tserver.avail.max.wait", "0",
```

Comment:
I am wondering of 0 is an appropriate default.  Perhaps a wait cycle of 1 minute as a default is more appropriate.  Just a thought.

Reply:
The idea was the default would be the current behavior so that no blocking would occur unless explicitly set. If this only goes to 2.x+, then a more appropriate default could be used, but for 1.9.x, didn't want to change the way that it acts now.